src/components/MoonpayModal/MoonpayModal.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { CustomModal } from 'components';
+
+interface MoonpayModalProps {
+  open: boolean;
+  onClose: () => void;
+}
+
+const MoonpayModal: React.FC<MoonpayModalProps> = ({ open, onClose }) => {
+  return (
+    <CustomModal
+      open={open}
+      onClose={onClose}
+      background='#fff'
+      overflow='hidden'
+    >
+      <div style={{ height: '100%', width: '100%', overflowY: 'auto' }}>
+        <iframe
+          title='moonpay'
+          allow='accelerometer; autoplay; camera; gyroscope; payment'
+          frameBorder='0'
+          height='600px'
+          src='https://buy.moonpay.com?apiKey=pk_live_72EGoobLnlgc8WB7AaxyiWu2S43dj8eY'
```

Comment:
The API key should not be hardcoded if should come from an environment file

Reply:
Added moonpay api key to env file.